rs/protobuf/def/types/v1/idkg.proto

```diff
@@ -233,6 +234,13 @@ message SchnorrSigShare {
   bytes sig_share_raw = 3;
 }
 
+message VetKdShare {
```

Comment:
Did you consider calling this `VetKdEncryptedKeyShare`? Or was it intentional that it has a different name from `ic_types::crypto::vetkd::VetKdEncryptedKeyShare`?

Another option would be `VetKdKeyShare`, which would be more in line with what we have for the ECDSA and Schnorr signature shares, as the related types contain `Sig`/`_sig_` in the name.

In any case, no strong opinion on the name, so I'm also OK with `VetKdShare` if you prefer that.

Reply:
I do like that it is different from the internal type. I think I will go for `VetKdKeyShare` as you suggest

Reply:
👍